### ISSUE TITLE
Error connection certificate

### ISSUE THREAD

**Issue by rubik12023** [mongodb-developer/pymongo-fastapi-crud#3]
Hi, 

I am trying to connect but I get a certificate error:

I have solved: main.py

    app.mongodb_client = MongoClient(config["ATLAS_URI"], ssl=True, ssl_cert_reqs=ssl.CERT_NONE)
    app.database = app.mongodb_client[config["DB_NAME"]]

**Comment by yoandysse** [comment]
> Hi,
> 
> I am trying to connect but I get a certificate error:
> 
> I have solved: main.py
> 
> ```
> app.mongodb_client = MongoClient(config["ATLAS_URI"], ssl=True, ssl_cert_reqs=ssl.CERT_NONE)
> app.database = app.mongodb_client[config["DB_NAME"]]
> ```

Yes. This resolve the problem, but i don't recommend it for code in production.

I recomend you https://github.com/mongodb-developer/pymongo-fastapi-crud/issues/1#issuecomment-1474819318

And try install pymongo[srv]
